autopilot/hostfilter.go

```diff
@@ -83,6 +85,10 @@ func isUpForRenewal(cfg Config, c rhpv2.Contract, blockHeight uint64) bool {
 	return blockHeight+cfg.Contracts.RenewWindow/2 >= c.EndHeight()
 }
 
+func isLowScore(cfg Config, h Host, threshold float64) bool {
+	return hostScore(cfg, h) < threshold
```

Comment:
I'd rather inline this. Having all the checks start with `is` is cute, but ultimately doesn't add much value (and requires you to add an indirection, or phrase something awkwardly, to fit the pattern)